pages/06.policies/09.Promoting-integrations/docs.md

```diff
@@ -0,0 +1,74 @@
+---
+title: '<your title>'
+twitterenable: true
+twittercardoptions: summary
+articleenabled: false
+orga: {  }
+personenabled: false
+facebookenable: true
+menu: 'Promoting integrations'
+taxonomy:
+    category:
+        - docs
+---
+
+# Introduction
+
+The Mautic website serves as a hub for the Mautic community, showcasing innovative projects that align with our mission and values. The purpose of this article is to provide clarity on the criteria and process for listing projects and integrations on the Mautic website's home page.
+
+# Criteria for Inclusion
+
+Projects featured on the Mautic website home page are selected based on their popularity and relevance to Mautic users. The decision to list a project is primarily made by the marketing team, considering factors such as the project's impact, use cases, and overall community value.
```

Comment:
The three were picked at random.
The point on quaterly poll is a good suggestion and we can do that.

Reply:
@RCheesley I can speak for CyberPanel. We were one of the first control panels who integrated Mautic one-click installation. 

Not only that we promoted and brought many users to Mautic. Then we've created lots of study material which have been seen by many people

https://www.youtube.com/watch?v=zeUfl0G6uOo&pp=ygURbWF1dGljIGN5YmVycGFuZWw%3D
https://www.youtube.com/watch?v=ML6NAE2h0xY&pp=ygURbWF1dGljIGN5YmVycGFuZWw%3D

Reply:
```suggestion
Projects featured in the Mautic Integrations Directory must be proposed by a member of Mautic via the [Community Portal](https://community.mautic.org/assemblies/marketing-team/f/53/proposals/new). Integrations require a minimum of five votes from the community to be considered for inclusion, and be endorsed by the Marketing Team. 

The projects listed on the Mautic website home page are selected based on their popularity and relevance to Mautic users, which is accomplished with a quarterly vote on the Community Portal of all eligible integrations. Members of Mautic are eligible to vote. 
```

@tobsowo I mean this should be documented. I believe we should use the community voting option. This is less open to personal biases and gives the community the opportunity to be engaged in the process and therefore it be more representative of sentiment. I'd suggest we limit it to members of Mautic only who are allowed to both propose integrations and vote. If quarterly proves to be too frequent we could make it 6-monthly.

I can help setting up the voting process and documenting it for you so that it's easy for you to reproduce every 3 months.